jsbuddy/covid19-estimator
# ft: implement covid19 estimator

#### What does this PR do?
This PR implements covid19 estimates for both impact and severeImpact

#### Description of Task to be completed?
- create utility functions
- implement estimator

#### How should this be manually tested?
Call function exported (default export) in src/estimator.js with expected input data as argument
